src/main/scala/is/hail/sparkextras/RepartitionedOrderedRDD2.scala

```diff
@@ -0,0 +1,69 @@
+package is.hail.sparkextras
+
+import is.hail.annotations._
+import is.hail.rvd.{OrderedRVD, OrderedRVDPartitioner, OrderedRVDType}
+import is.hail.utils._
+import org.apache.spark._
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.Row
+
+/**
+  * Repartition prev to comply with newPartitioner, using narrow dependencies.
+  * Assumes new key type is a prefix of old key type, so no reordering is
+  * needed. No assumption should need to be made about partition keys, but currently
+  * assumes old partition key type is a prefix of the new partition key type.
+  */
+class RepartitionedOrderedRDD2(
```

Comment:
RepartitionedOrderedRVD?

Reply:
This is an `RDD[RegionValue]`, not an `RVD`.